lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -151,24 +151,24 @@ bool SessionStateEnforcer::m5g_update_session_context(
 }
 
 uint32_t SessionStateEnforcer::update_session_rules(
-    const std::string& imsi, std::unique_ptr<SessionState>& session_state,
-    bool gnb_teid_get, bool upf_teid_get,
-    SessionStateUpdateCriteria* session_uc) {
+    std::unique_ptr<SessionState>& session, bool gnb_teid_get,

```

Comment:
This is not related to this PR, so feel free to just add a TODO, but 
`gnb_teid_get` -> `get_gnb_teid`
`upf_teid_get ` -> `get_upf_teid`
so that it reads more like english.

Reply:
Updated the parameters as suggested.
gnb_teid_get -> get_gnb_teid
upf_teid_get -> get_upf_teid